browser-test/src/support/applicant_questions.ts

```diff
@@ -343,11 +343,21 @@ export class ApplicantQuestions {
     await waitForPageJsLoad(this.page)
   }
 
+  async clickConfirmAddress() {
+    await this.page.click('text="Confirm address"')
+    await waitForPageJsLoad(this.page)
+  }
+
   async clickEdit() {
     await this.page.click('text="Edit"')
     await waitForPageJsLoad(this.page)
   }
 
+  async clickGoBackAndEdit() {
+    await this.page.click('text="Go back and edit"')
```

Comment:
```suggestion
    await this.page.getByRole('button', { name: 'Go back and edit' }).click()
```

(blocking) See notes above